packages/cloudflare/README.md

```diff
@@ -70,7 +70,6 @@ export default config;
 
 - Next cache is not supported in the experimental branch yet
```

Comment:
This can be removed ( or changed in the cache PR as well )

Reply:
^that should actually go in the #194